documentation/api/query/tutorial-pql.markdown

```diff
@@ -30,7 +30,32 @@ and specifying a `query` URL parameter (in the GET case) or a JSON-valued
 payload in the POST case, which contains the query to execute. Results are
 always returned in `application/json` form.
 
-Queries are usually issued from code, but you can easily issue them from the command line by using curl.
+Queries are usually issued from code, but you can easily issue them from the
+command line by using the [PuppetDB CLI][cli_install] or using [curl][curl].
+
+### Querying with the PuppetDB CLI
+
+[See the PuppetDB CLI installation page for more information about using the PuppetDB CLI.][cli_install]
+
+**Without SSL:**
+
+    puppet query '<PQL query>' \
+      --urls http://puppetdb.example.com:8080/pdb/query/v4
```

Comment:
Oh this is repeated in the pql reference, hohum - I had a feeling those two docs would overlap.